extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -81,7 +81,13 @@ iframe.i-amphtml-consent-ui-fill {
   border: none;
 }
 
+/**
+ * CMP Modal class appolied when asking for CMP consent.
+ * See: https://developers.google.com/web/updates/2016/12/url-bar-resizing
```

Comment:
Thank you @torch2424, the comment is very useful! 

I have one question after reading the doc.  I saw 
> There is one exception to the above changes and that is for elements that are position: fixed. Their behavior remains unchanged. That is, a position: fixed element whose containing block is the ICB will resize in response to the URL bar showing or hiding.

Do we still need to change 100% to 100vh if we apply `position: fixed` here?

Reply:
Yes, this is because even though the viewport height and percentages are the same end value, during the transition, viewport units will be calculated, but percentage based values will jump from the before and after.

Try the demo they linked in the article: https://bokand.github.io/demo/urlbarsize.html

And you will see that when the url bar hides, the percentage based vertical div will jump sizes, instead of being consistent 😄